includes/class-wc-ajax.php

```diff
@@ -978,33 +978,8 @@ public static function add_order_fee() {
 				throw new Exception( __( 'Invalid order', 'woocommerce' ) );
 			}
 
-			$amount = isset( $_POST['amount'] ) ? wc_clean( wp_unslash( $_POST['amount'] ) ) : 0;
```

Comment:
Sorry, could you give a little context around the decision to remove all of this?